presto-docs/src/main/sphinx/release/release-0.219.rst

```diff
@@ -0,0 +1,62 @@
+=============
+Release 0.219
+=============
+
+General Changes
+---------------
+
+* Fix a correctness bug for queries with a ``LIMIT`` on top of a ``FULL OUTER JOIN``.
+* Fix a correctness bug in how word boundaries are handled for Joni regular expression functions.
```

Comment:
What does "Joni regular expression function" mean? Should we say `for regular expression functions when using the Joni regex engine` or sth along those lines to clarify a bit.

Reply:
Do we need to specify `Joni` here? Should we just say `regular expression functions`?

Reply:
@nezihyigitbasi @rschlussel Just to clarify, are user able to choose which regex engine to use?

Reply:
User can select from joni, re2j with the `regex-library` config, and this is only broken for joni, so we should mention the engine here.

Reply:
@mayankgarg1990 I think specifying `Joni` here is more appropriate for precision.